docs/Parameters-Tuning.rst

```diff
@@ -70,6 +70,7 @@ LightGBM adds nodes to trees based on the gain from adding that node, regardless
 
 .. image:: ./_static/images/leaf-wise.png
    :align: center
+   :alt: Figure of Leaf-wise tree growth from Features documentation which grows trees leaf-wise (best-first)
```

Comment:
I don't think this would be very clear for someone who cannot already see the image. Alternative text is intended to describe an image for someone who cannot see it, either because the image cannot be loaded or because the person uses a screen reader to read web pages.

What do you think about this instead?

```suggestion
   :alt: Three consecutive images of decision trees, where each shows the tree with an additional two leaf nodes added. Shows that leaf-wise growth can result in trees that have some branches which are longer than other.
```

Reply:
Ok I thought that a too long alt text may not be allowed else your suggestion is perfect for explaining the image

Reply:
Should I commit the same??

Reply:
Yes, if you agree with the suggestion you can click "Commit suggestion" right here in the GitHub user interface. That way, you don't have to manually change local files and push a new commit.